backpack-compose/src/main/kotlin/net/skyscanner/backpack/compose/tokens/BpkLineHeight.kt

```diff
@@ -18,15 +18,27 @@ import androidx.compose.ui.unit.TextUnit
 import androidx.compose.ui.unit.sp
 
 internal object BpkLineHeight {
-  internal val Base: TextUnit = 19.0.sp
+  internal val `5xl`: TextUnit = 64.0.sp
 
-  internal val Lg: TextUnit = 23.0.sp
+  internal val `6xl`: TextUnit = 76.0.sp
 
-  internal val Sm: TextUnit = 16.0.sp
+  internal val `7xl`: TextUnit = 96.0.sp
 
-  internal val Xl: TextUnit = 28.0.sp
+  internal val `8xl`: TextUnit = 120.0.sp
```

Comment:
We may need to ask the designers to change thouse names, it may be causing dev experience issues on Android and some other issues on other platform.

Reply:
We specifically wanted to avoid naming them xxxxxxxxl to avoid mistyping the number of x's. These tokens will also only be internal to backpack and not external to the app - only the styles will which don't contain numbers.